src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -81,17 +81,21 @@ trait IterableOps[+A] extends Any {
   /** A view representing the elements of this collection. */
   def view: View[A] = View.fromIterator(iterator())
 
-  /** Given a collection factory `fi` for collections of type constructor `C`,
-    *  convert this collection to one of type `C[A]`. Example uses:
+  /** Given a collection factory `fi`, convert this collection to the appropriate
+    * representation for the current element type `A`. Example uses:
     *
     *      xs.to(List)
     *      xs.to(ArrayBuffer)
+    *      xs.to(BitSet) // for xs: Iterable[Int]
     */
-  def to[C[X] <: Iterable[X]](fi: FromIterable[C]): C[A @uncheckedVariance] =
+  def to[F <: FromIterableBase[A]](fi: F): F#To[A @uncheckedVariance] =
```

Comment:
Ah, right, this looked safe enough for me but I haven't tested it on Dotty. It's pretty much the same as https://github.com/lampepfl/dotty/pull/1051#issuecomment-181577039 though, so it won't work. I'll change it to use the path-dependent type instead.